lib/bank/savings_account.rb

```diff
@@ -0,0 +1,30 @@
+module Bank
+  class SavingsAccount < Account
+
+    def initialize(id, initial_balance)
+      raise ArgumentError, "Balance must be a minimum of $10." if initial_balance < 10
+
+      @id = id
+      @balance = initial_balance.to_i
+      @minimum_balance = 12
```

Comment:
I did it as a quick way to make sure the withdrawal could be made (including the $2 fee) but I rewrote how this was handle and like the new solution much better. Thanks!